.github/workflows/post-pr-cleanup.yml

```
@@ -0,0 +1,61 @@
+name: cleanup-codecov-reports
+
+
+# This workflow deletes the html reports for the given PR to tidy up the helper repository
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
+
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: dopeshot/beyond-life-codecov
+          token: ${{ secrets.BOT_PAT }}
+          ref: main
+
+      - name: Extract branch name
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/} | tr '/' '_')"
+        id: extract_branch
+
+      - name: cleanup html reports
+        run: rm -rf  ${{ steps.extract_branch.outputs.branch }} || true
+
+      - name: Commit
+        run: |
+          git config user.name ${{ secrets.BOT_NAME }}
+          git config user.email ${{ secrets.BOT_MAIL }}
+          git add ./${{ steps.extract_branch.outputs.branch }}/${{ inputs.context }} 
+          git commit -m "${{ inputs.context }} coverage for branch ${{ steps.extract_branch.outputs.branch}}" || echo "No changes to commit"
+
+      - name: Push
+        run: git push
+
+
+
+  update-pr-comment:
+    runs-on: ubuntu-latest
+    needs: cleanup
+    strategy:
+      matrix:
+        context:
+          - backend
+          - frontend
+    steps:
+
+      - name: Comment coverage on PR
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+            This PR has been closed. All codecov information has been deleted...

```

Vielleicht branch noch hinzufügen, das es komplett klar ist? (auch wenn er im PR drin ist) 
```suggestion
        This PR has been closed. All codecov information for branch ${{ steps.extract_branch.outputs.branch }} has been deleted.
```